less/variables.less

```diff
@@ -1 +1,22 @@
-// Variables
\ No newline at end of file
+// Variables
+@font-color: @shark;
+@button-bg: @shark;
```

Comment:
Would like to hear your explanation for using multiple variables for one value 👂